docs/running/server/wes.rst

```diff
@@ -75,6 +75,9 @@ Below is a detailed summary of all available options:
             Specify the default parameters to be sent to the workflow engine for each run.  Accepts multiple values.
 
             Example: ``toil server --opt=--logLevel=CRITICAL --opt=--workDir=/tmp``.
+--dest_bucket_base DEST_BUCKET_BASE
+            Direct CWL workflows to save output files to dynamically generated unique paths under the given URL.
+            Supports AWS S3.
```

Comment:
I don't want it to be just `--dest_bucket`, because that's the same as the CWL runner option, while the value is not passed unmodified to the CWL runner. It would be confusing to use the same name but end up eventually running on a different value.